contrib/spring_cloud/core/src/main/java/io/opencensus/contrib/springcloud/instrument/web/HttpServletFilter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.springcloud.instrument.web;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import io.opencensus.contrib.http.servlet.OcHttpServletFilter;
+import javax.servlet.Filter;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+@Component
+@Order(1)
```

Comment:
I used +10 to leave some room for other important filter (may be security related) to precede tracing/monitoring. +10 may be too much. do you have a recommendation here?

Reply:
@rghetia we had similar problem in Application insights and there was a discussion around it.

You may want to refer https://github.com/Microsoft/ApplicationInsights-Java/pull/766#issuecomment-433195169 for more relevant details.

in short there are some default springboot filters which are responsible for setting error codes, and get the correct timing. It would be very hard to get that details if the filter lies below that in the chain. Therefore HIGHEST_PRECEDENCE is needed imo. But, I am open if you find better alternatives

Reply:
I would recommend the exact same value that Springs Tracing uses for their filter, unless you expect them both to be there, then maybe an additional `+ 1`

```
static final int TRACING_FILTER_ORDER = Ordered.HIGHEST_PRECEDENCE + 5;
```

Reply:
SG. I'll make the change.

Reply:
@devinsba yes your suggestion looks more apt.